### ISSUE TITLE
macOS-macos Clipboard!

### ISSUE THREAD

**Issue by arozhankov** [utmapp/UTM#6901]
What's going on with the clipboard? it works, it doesn't work. The causes of the problem are random. I have never determined when or at what point the clipboard disconnects.

Guest: macos 15.1
Host: macos 15.2

Seems like a simple task, but the macos-macos clipboard problem has been going on for years at utm

**Configuration**  
* UTM Version: 4.6.3
* macOS Version: 15.2
* Mac Chip (Intel, M1, ...): M3Pro

**Comment by unizhu** [comment]
yes... the clipboard is broken for most of the time. I use nc command in terminal to send messages between 2 MacOS

**Comment by osy** [comment]
> Seems like a simple task, but the macos-macos clipboard problem has been going on for years at utm

As an open source project, we invite you and anyone else to contribute fixes for such simple tasks :)

**Comment by erikschul** [comment]
My experience with debian and nixos is that the clipboard works from MacOS to the guest, but not from the guest to MacOS.
This is with spice-vdagentd running on the system, and spice-vdagent running in the user session.

Maybe there is a signal from spice-vdagent that isn't being received/processed in UTM to set the clipboard? Or MacOS app permission to set the clipboard?
